tools/astarte_import/README.md

```diff
@@ -8,48 +8,334 @@ docker run -e CASSANDRA_DB_HOST=127.0.0.1 -e CASSANDRA_DB_PORT=9042 \
-          <datastream path="/realValues">
-            <object reception_timestamp="2019-06-11T13:24:03.200820Z">
-              <item name="/x">0.1</item>
-              <item name="/y">0.2</item>
-              <item name="/z">0.3</item>
-            </object>
-            <object reception_timestamp="2019-06-11T13:26:28.994144Z">
-              <item name="/x">1.0</item>
-              <item name="/y">2.0</item>
-              <item name="/z">3.0</item>
-            </object>
-            <object reception_timestamp="2019-06-11T13:26:44.218092Z">
-              <item name="/x">10</item>
-              <item name="/y">20</item>
-              <item name="/z">30</item>
-            </object>
-          </datastream>
-        </interface>
-        <interface name="org.astarteplatform.PropertyValue" major_version="0" minor_version="1" active="true">
-          <property path="/realValue" reception_timestamp="2019-06-12T14:45:49.706034Z">4.2</property>
-        </interface>
-      </interfaces>
-    </device>
-  </devices>
+<devices>
+<device device_id="yKA3CMd07kWaDyj6aMP4Dg">
+  <protocol pending_empty_cache="false" revision="0"></protocol>
+  <registration first_registration="2024-05-30T13:49:57.045Z" secret_bcrypt_hash="$2b$12$bKly9EEKmxfVyDeXjXu1vOebWgr34C8r4IHd9Cd.34Ozm0TWVo1Ve"></registration>
+  <credentials cert_aki="a8eaf08a797f0b10bb9e7b5dca027ec2571c5ea6" cert_serial="324725654494785828109237459525026742139358888604" first_credentials_request="2024-05-30T13:49:57.355Z" inhibit_request="false"></credentials>
+  <stats last_connection="2024-05-30T13:49:57.561Z" last_disconnection="2024-05-30T13:51:00.038Z" last_seen_ip="198.51.100.89" total_received_bytes="3960" total_received_msgs="64"></stats>
+  <interfaces>
+    <interface active="true" major_version="0" minor_version="1" name="test.individual.parametric.Datastream">
+      <datastream path="/a/boolean">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">true</value>
+      </datastream>
+      <datastream path="/a/integer">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">123</value>
+      </datastream>
+      <datastream path="/a/double">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">123.45</value>
+      </datastream>
+      <datastream path="/a/longinteger">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">123456789012345</value>
+      </datastream>
+      <datastream path="/a/string">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">example string</value>
+      </datastream>
+      <datastream path="/a/binaryblob">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">aGVsbG8gd29ybGQ=</value>
+      </datastream>
+      <datastream path="/a/datetime">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">2024-10-17T13:25:19.130Z</value>
+      </datastream>
+      <datastream path="/a/doublearray">
+        <value reception_timestamp="2024-09-09T09:00:00.000Z">[123.45, 678.90]</value>
```

Comment:
Here (and in the following lines regarding arrays) XML and JSON are mixed. While this is technically valid, an inconsistency in the representation of data is introduced. I would suggest to use this representation instead, encoding arrays as lists of XML elements:
```suggestion
        <value reception_timestamp="2024-09-09T09:00:00.000Z">
          <element>123.45</element>
          <element>678.9</element>
        </value>
```

Reply:
Should we make the same changes in the Property and Object interfaces? For example:
``` xml
<property path="/a/doublearray" reception_timestamp="2024-09-04T11:31:36.956Z">
       <element>123.45</element>
       <element>678.90</element>
</property>
```
``` xml
<item name="/doublearray">
      <element>123.45</element>
      <element>678.90</element>
</item>
```

Reply:
Yes @osmanhadzic , good catch!